runtime/lua/vim/lsp/util.lua

```diff
@@ -110,6 +110,16 @@ local function split_lines(value)
   return split(value, '\n', true)
 end
 
+---@private
+local function create_window_without_focus()
+  local prev = vim.api.nvim_get_current_win()
+  -- TODO respect splitright/splitbelow
+  vim.cmd(':noautocmd new')
+  local new = vim.api.nvim_get_current_win()
+  vim.api.nvim_set_current_win(prev)
+  return new
+end
+
```

Comment:
Is there a better way to do this?

Reply:
Also, how do we respect splitright/splitbelow? on `nvim -u NONE` they both default to true, and changing it doesn't seem to affect `:new`.

Reply:
`:new` should consider them, it shouldn't need any special handling. 

Is there a reason for the `:noautocmd` in the `new` call? Not sure if that is needed.

Otherwise this seems fine

Reply:
Not really, seemed like a safe choice for 'not focus'. Should I remove it?